clap_complete/tests/snapshots/quoting.zsh

```diff
@@ -17,10 +17,10 @@ _my-app() {
     _arguments "${_arguments_options[@]}" \
 '--single-quotes[Can be '\''always'\'', '\''auto'\'', or '\''never'\'']' \
 '--double-quotes[Can be "always", "auto", or "never"]' \
-'--backticks[For more information see `echo test`]' \
+'--backticks[For more information see \`echo test\`]' \
```

Comment:
Can you squash this commit into the other one as these aren't atomic. @

Reply:
Updated.